Tests/XCUITests/SmokeXCUITests.xctestplan

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
We will need to move also the other testplans:
Smoketest2, Smoketest3, Smoketest4, each one with the tests that have enabled